src/main/java/org/spring/FullVersion/Family.java

```diff
@@ -1,39 +1,37 @@
 package org.spring.FullVersion;
 
-import java.util.Arrays;
-import java.util.Objects;
-import java.util.Random;
+import java.util.*;
```

Comment:
pls avoid using import with wildcard (*)
You can read:
[](https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad#:~:text=739,your%20clarity%20otherwise.)

and how to fix pls investigate)